plugins/inputs/icinga2/README.md

```diff
@@ -25,8 +25,10 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## Required Icinga2 server address
   # server = "https://localhost:5665"
   
-  ## Required Icinga2 object type ("services" or "hosts")
-  # object_type = "services"
+  ## Icinga2 object type ("services" or "hosts") from /v1/objects endpoint
+  # objects = ["services", "hosts"]
+  ## Metrics from /v1/status endpoint
+  # status = ["ApiListener", "CIB", "IdoMysqlConnection", "IdoPgsqlConnection"]
```

Comment:
The values in the config file are the defaults. As such this should be empty and you can add these as an example.

Reply:
Sure, I've updated the config.